### DISCUSSION TITLE
Shlink Installation: 404 Error on Access

### DISCUSSION THREAD

**Discussion by You-Tribe** [runtipi/runtipi-appstore#4605] [Q&A]
Hi @steveiliop56,
There seems to be a problem when installing the app, I've already had the problem with other apps in the store.
I get a 404 error (cf. short.chedmatic.fr)
Do you have a solution or can you fix the problem if it's global?
Thanks :)

**Comment by steveiliop56** [comment]
Hello,

If I remember correctly shlink doesn't have a builtin web ui.
